src/server/session.ts

```diff
@@ -2703,11 +2705,33 @@ export class Session<TMessage = string> implements EventSender {
                 const renameScriptInfo = project.getScriptInfoForNormalizedPath(toNormalizedPath(renameFilename))!;
                 mappedRenameLocation = getLocationInNewDocument(getSnapshotText(renameScriptInfo.getSnapshot()), renameFilename, renameLocation, edits);
             }
-            return { renameLocation: mappedRenameLocation, renameFilename, edits: this.mapTextChangesToCodeEdits(edits) };
+            return {
+                renameLocation: mappedRenameLocation,
+                renameFilename,
+                edits: this.mapTextChangesToCodeEdits(edits)
+            };
         }
-        else {
-            return result;
+        return result;
+    }
+
+    private getMoveToRefactoringFileSuggestions(args: protocol.GetMoveToRefactoringFileSuggestionsRequestArgs): { newFilename: string, files: string[] }{
+        const { file, project } = this.getFileAndProject(args);
+        const allFiles: string[] = [];
+        let fileName = "";
+        updateProjectIfDirty(project);

```

Comment:
```suggestion
```

you dont need to update thrp orject as that will happen when you call `getLangaugeService`